packages/render-dedupe-check-issues/README.md

```diff
@@ -0,0 +1,13 @@
+# @pnpm/render-dedupe-check-issues
```

Comment:
Don't put a new package to the packages directory. It is hard to find packages if all are in one directory. Maybe put it to the cli directory if there is no more specific scope for it. Then name it `@pnpm/cli.render-dedupe-check-issues`

Reply:
Sounds good! I don't see a better scope for it. Will move to `@pnpm/cli.render-dedupe-check-issues`.